rir/src/compiler/pir/type.h

```diff
@@ -174,7 +176,8 @@ struct PirType {
     static PirType val() {
         return PirType(vecs() | list() | RType::sym | RType::chr | RType::raw |
                        RType::closure | RType::prom | RType::code | RType::env |
-                       RType::missing | RType::unbound | RType::ast);
+                       RType::missing | RType::unbound | RType::ast)
+            .nonLazy();
```

Comment:
same comment as bellow: this `nonLazy` should not be neccessary. if this combination returns a lazy value, then the constructor needs to be fixed to not return a lazy value.